src/problem/constr/constr_NBOUND.c

```diff
@@ -598,8 +598,8 @@ void CONSTR_NBOUND_eval_step(Constr* c, Branch* br, int t, Vec* values) {
 	  BUS_has_flags(bus,FLAG_VARS,BUS_VAR_VMAG)) {
 
 	u = VEC_get(values,BUS_get_index_v_mag(bus,t));
-	umax = BUS_get_v_max(bus);
-	umin = BUS_get_v_min(bus);
+	umax = BUS_get_v_max_reg(bus);
+	umin = BUS_get_v_min_reg(bus);
```

Comment:
I think so. The reason is that these are limits on variables and have no direct association with regulation. For example, the constraints LBOUND or NBOUND may be used to enforce voltage magnitude limits in OPF, so why should they be based on regulation limits? In power flow, only constraints that enforce voltage regulation should impose these regulation limits. This is what I think makes the most sense. Only constraints that directly deal with voltage regulation should use reg limits. The others should not. Let me know if you think differently. At least I think this should be the "default" behavior.